tests/bruno/e2e/invoice-ops/1-gen-invoice-err.bru

```diff
@@ -0,0 +1,42 @@
+meta {
+  name: connect peer
+  type: http
+  seq: 1
```

Comment:
Judging from the amount of duplicated code and duplicated seq number here (seq should be automatically incremental), I think you are copying the files directly, you may try to play with https://www.usebruno.com/ the app and right click on this file and select clone (this will automatically increment the seq number). It was the first time I use this app, and it's amazing.